Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "tikv-client"
+version = "0.0.0"
+keywords = ["TiKV", "KV", "distributed-systems"]
+license = "Apache-2.0"
+authors = ["The TiKV Project Developers"]
+repository = "https://github.com/tikv/client-rust"
+description = "The rust language implementation of TiKV client."
+
+[lib]
+name = "tikv_client"
```

Comment:
mostly, we name our Go library as ticlient, maybe we can use the same name here. :-)

Reply:
/cc @c4pt0r

Reply:
I had a little concern what if there are other storage system built on top of TiKV and we want to implement a Rust client for it as well?

Reply:
Although it's probably too early to say that. But I really pictured a day that we could build a distributed file system or maybe timeseries database on top of TiKV. And for the nature of it's protocol complexity, it's likely a dedicated client is preferred.

Reply:
Can't believe that after more than a decade, HDFS probably wins in ecosystem but still loses in scalability due to HA NameNode architecture. Even Google's initial GFS has something better than HDFS. Although there are federation support but it sounds better to have a clustered NameNode natively which can be built on top of TiKV.

Reply:
@siddontang I can only see `TiClient` used twice in TiDB's code: https://github.com/pingcap/tidb/search?q=TiClient&unscoped_q=TiClient.

Also in the name of one test file https://github.com/pingcap/tidb/blob/cb03f2bec151033a38dfd17715a42cf3728b0541/store/tikv/ticlient_test.go

I think using a different name than `tikv_client` risks the user not understanding what this is for. `ticlient` is ambiguous and could mean `tidb` or `tispark` or something else.